safe_transaction_service/history/indexers/ethereum_indexer.py

```diff
@@ -146,7 +146,7 @@ def get_block_numbers_for_search(
             current_block_number - self.confirmations
         ):
             # Reindex again when it's almost synced to prevent reorg/missing elements issues
-            from_block_number = max(from_block_number - self.blocks_to_reindex_again, 0)
+            from_block_number = max(from_block_number - self.blocks_to_reindex_again, 1)
```

Comment:
@Uxio0 I'm not sure about this, the issue that I found was that sometimes the indexer start in the block 1, and only in cases where the `block_process_limit` was bigger than `current_block` the indexer starts from 0.

Reply:
Block 0 is valid, so we should always start in block 0: https://etherscan.io/block/0